File: bmi/bmi.cpp
#include <iostream>
#include <cmath>
#include <iomanip> //untuk std::format
int main() {
	std::cout<<"Kalkulator BMI\n";
	std::cout<<"==============\n";
	float berat;
	float tinggi;
	std::cout<<"Masukan berat (dalam kg) :";
	std::cin>>berat;
	std::cout<<"Masukan tinggi (dalam m) :";
	std::cin>>tinggi;
	if (berat == 0 || tinggi == 0) {
		std::cout<<"Nilai tidak boleh nol\n";
		return 0;
	}
	int bmi = berat/pow(tinggi,2);
	std::cout<<"Score BMI mu "<< std::fixed << std::setprecision(2) << bmi <<"\n";
	std::string status;
	if (bmi < 18.5) {
		status = "underweight";
	} else if (bmi < 24.9) {
		status = "ideal";
	} else if (bmi < 29.9) {
		status = "overweight";
	} else if (bmi > 29.9) {
		status = "obesitas";
	}
	std::cout<<"Kamu termasuk "<<status<<"\n";
	return 0;
}

File: bmi/bmi.lua
print("Kalkulator BMI")
print("==============")
io.write("Masukan berat badan (dalam kg) :")
local berat = tonumber(io.read())
io.write("Masukan tinggi badan (dalam m) :")
local tinggi = tonumber(io.read())
if berat == 0 or tinggi == 0 then
	print("Nilai tidak boleh nol")
else
	local bmi = berat/tinggi^2
	print("Score bmi anda "..string.format("%.2f", bmi))
	local status
	if bmi<18.5 then
		status = "underweight"
	elseif bmi<24.9 then
		status = "ideal"
	elseif bmi<29.9 then
		status = "overweight"
	else
		status = "obesitas"
	end
	print("kamu termasuk "..status)
end
